ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+***PCSX2 version:***
+``(For example, stable build v1.4.0 (or) development builds v1.5.xxx)``
+
+- **Note**: It's preferred to only report bugs which are present on the [latest development builds](http://pcsx2.net/download/development.html).
+
+
+***PCSX2 options:***
+``(For example, Clamping/Rounding, Speedhacks, GameFixes)``
+
+- **Note**: Generally it's recommended to have settings as default as possible and use ``Safest`` preset along with MTVU hack disabled. If there's no issue with these settings but there is an issue with some other non-default option, then please state it explicitly.
+
+
+***Plugins used:***
+``(Relevant/Non-default settings of GS/PAD/SPU2/CDVD/USB plugins)``
```

Comment:
Well for me it is important. If the bug is only the 
* DX HW renderer => don't care ;)
* GL HW renderer, might worth to fix it and I can use the SW renderer as reference
* SW renderer. Complex bug that will need attention. Or core bug.

So the test of all renderers can help to resolve the bug.